test/libmpv_lifetime_suppr.txt

```diff
@@ -0,0 +1 @@
+leak:unknown module
```

Comment:
It is not correct. unknown module is any unknown module, maybe it was unloaded or asan lost the reference to it, but this most likely is valid leak, that happens from uninstrumented code and/or unloaded module.